package.json

```diff
@@ -45,61 +51,70 @@
     "prop-types": "15.8.1",
     "qrcode.react": "3.1.0",
     "react": "18.2.0",
+    "react-app-rewired": "2.2.1",
     "react-copy-to-clipboard": "5.1.0",
     "react-dom": "18.2.0",
     "react-loading": "2.0.3",
     "react-paginate": "8.2.0",
     "react-redux": "7.2.8",
     "react-router-dom": "6.22.3",
-    "react-scripts": "4.0.3",
+    "react-scripts": "5.0.1",
     "redux-saga": "1.2.1",
     "redux-thunk": "2.4.1",
     "ttag": "1.8.6",
     "unleash-proxy-client": "3.2.0",
-    "viz.js": "2.1.2"
+    "viz.js": "2.1.2",
+    "node-stdlib-browser": "1.2.0"
```

Comment:
Do we need it here? Or just in the `devDependencies`? I'm asking just because you also added it there.